molgenis-api/src/main/java/org/molgenis/api/VersionApiNamespace.java

```diff
@@ -0,0 +1,8 @@
+package org.molgenis.api;
+
+public class VersionApiNamespace {
```

Comment:
We agreed on ```OPTIONS /api``` to expose the application version, right? That would make this class unnecessary.